packages/site-admin/src/components/site-icon/stories/style.stories.scss

```diff
@@ -0,0 +1,8 @@
+.story-site-icon {
+	width: 130px;
+
+	.site-admin-site-icon__image {
+		aspect-ratio: 1 / 1;
+		object-fit: contain;
+	}
```

Comment:
Overriding an internal classname like this is probably not something we want to showcase here, as it is bad practice.